hedera-node/src/test/java/com/hedera/services/config/Log4JExploitTest.java

```diff
@@ -57,19 +58,53 @@ public static void restoreStreams() {
 			"configuration/dev/log4j2.xml",
 			"configuration/compose/log4j2.xml"})
 	void testLog4jExploit(final String configPath) {
+		final var malUri_toFailLookUp = "${jndi:https://previewnet.mirrornode.hedera.com/api/v1/accounts?account.id=0.0.90}";
+		final var expectedLog = "We’re doomed. " + malUri_toFailLookUp;
+		final var stackStraceSample = "at org.apache.logging.log4j.core.net.JndiManager.lookup";
+
 		LoggerContext context = (LoggerContext) LogManager.getContext(false);
 		File file = new File(configPath);
 		context.setConfigLocation(file.toURI());
 
+		LogManager.getLogger().warn(expectedLog);
+
+		// Test if the jndi lookup activated and failed because of bad naming.
+		var actualLog = outContent.toString();
+
+		assertTrue(actualLog.contains(expectedLog));
+		assertFalse(actualLog.contains(stackStraceSample));
+	}
+
+	@ParameterizedTest
+	@CsvSource({
+			"src/test/resources/log4j2-WithMDC-test.xml"
```

Comment:
I think it is fine to have the WithMDC test just to verify that the test itself is working, but you also need to test the other log4j files to make sure they *aren't* exploitable. Maybe have two params, the file and whether the file should pass or fail the check, and then in the test make sure it behaves appropriately.